pokedex/data/csv/ability_flavor_text.csv

```diff
@@ -17469,3 +17469,9 @@ les attaques super efficaces."
 威力が　さらに　上がる。"
 233,18,12,"效果绝佳的攻击，
 威力会变得更强。"
+20000,20,9,"Powers up Electric-type moves."
+20000,20,9,"Powers up Dragon-type moves."
+20000,20,9,"When the Pokémon knocks out a target, it utters a chilling neigh, which boosts its Attack stat."
+20000,20,9,"When the Pokémon knocks out a target, it utters a terrifying neigh, which boosts its Sp. Atk stat."
+20000,20,9,"This Ability combines the effects of both Calyrex's Unnerve Ability and Glastrier's Chilling Neigh Ability."
+20000,20,9,"This Ability combines the effects of both Calyrex's Unnerve Ability and Spectrier's Grim Neigh Ability."
```

Comment:
It's impossible that we have `20000` for every ability added. 

those 20000s should become 262-266.

Reply:
Hi @Naramsim - how would you like to handle the "As One" ability for Shadow/Ice Rider Calyrex formes? It's a weird situation - Bulbapedia has it listed as "266/267" since it's technically a combination of two different abilities, with two permutations: 266 (As One/Chilling Neigh) and 267 (As One/Grim Neigh). Please advise on how I should set that up - for the moment, I've added a "267" to the ability_flavor_text.csv.

Reply:
Hi @alex-whan, thanks! I would then duplicate the `As One` ability and add an identical 267 ID.

Reply:
In the `ability_names` and `abilities` CSV files.

Reply:
Hi @Naramsim, got it, thank you. I've created the duplicate listings for the As One ability for #267 and added them to the `ability_names` and `abilities` CSV files. Change is committed.